apps/heft/src/plugins/JestPlugin/jest-identity-mock-transform.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { InitialOptionsWithRootDir } from '@jest/types/build/Config';
+
+const code: string = [
+  "// This proxy is injected by Heft's jest-identity-mock-transform.  See Heft documentation for details.",
+  'const proxy = new Proxy({}, {',
+  '  get: function getter(target, key) {',
+  "    if (key === '__esModule') {",
+  '      return false;',
+  '    }',
+  '    return key;',
+  '  }',
+  '});',
+  'module.exports = proxy;'
```

Comment:
Might be neat to write this in TS and have this transform read that from disk and use that.

Reply:
Good idea -- fixed